pnpm-lock.yaml

```diff
@@ -45327,7 +45272,7 @@ packages:
     version: 0.1.0
     hasBin: true
     dependencies:
-      '@rushstack/node-core-library': 3.61.0
+      '@rushstack/node-core-library': 3.63.0
```

Comment:
Can you revert his change? I don't think this is required?

Reply:
Sure